bull-transformer-generator/transformer-generator-core/src/main/java/com/hotels/beans/generator/core/TransformerSpec.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beans.generator.core;
```

Comment:
You have a lot of Check Style violation, could you please fix them?

Reply:
As discussed in person, these errors are generated in my build as the license plugin will remove the blank line between license and package definition. I'll try to find a solution.